File: barseek-bar-mservice/src/main/java/com/example/barseek_bar_mservice/kafka/p_events/BarCreatedEvent.java
package com.example.barseek_bar_mservice.kafka.p_events;


import lombok.Builder;
import lombok.Data;

import java.util.UUID;

@Data
@Builder
public class BarCreatedEvent {
    private Long barId;
    private String name;
    private String address;
    private UUID ownerId;
}

File: barseek-review-microservice/pkg/kafkaAdapters/coEvents/BarCreatedEvent.go
package coevents

type BarCreatedEvent struct {
	BarID   int64  `json:"barId"`
	Name    string `json:"name"`
	Address string `json:"address"`
	OwnerID int64  `json:"ownerId"`
}

func NewBarCreatedEvent(barID int64, name, address string, ownerID int64) *BarCreatedEvent {
	return &BarCreatedEvent{
		BarID:   barID,
		Name:    name,
		Address: address,
		OwnerID: ownerID,
	}
}
